Doc/library/concurrent.futures.rst

```diff
@@ -21,6 +21,11 @@ or separate processes, using :class:`ProcessPoolExecutor`.
 Each implements the same interface, which is defined
 by the abstract :class:`Executor` class.
 
+:class:`concurrent.futures.Future` must not be confused with
+:class:`asyncio.Future`, which is designed for use with :mod:`asyncio`
+tasks and coroutines. See the :doc:`asyncio-future` documentation for a
+detailed comparison of the two.
```

Comment:
```suggestion
:class:`concurrent.futures.Future` must not be confused with
:class:`asyncio.Future`, which is designed for use with :mod:`asyncio`
tasks and coroutines. See the :doc:`asyncio's Future <asyncio-future>`
documentation for a detailed comparison of the two.
```

(otherwise it's rendered as "Future" only)

Reply:
<img width="1427" height="93" alt="image" src="https://github.com/user-attachments/assets/91c68e47-1d13-4c00-8db8-3eef80eb2d8d" />